config/policies.js

```diff
@@ -33,14 +33,15 @@ module.exports.policies = {
     status: true,
   },
 
-  'game/reconnect': [ 'isLoggedIn', 'isInGame' ],
   'game/create': [ 'isLoggedIn', 'hasGameName', 'hasNoProfanity' ],
   'game/get-list': 'isLoggedIn',
   'game/subscribe': [ 'isLoggedIn', 'hasGameId' ],
-  'game/spectate': [ 'isLoggedIn', 'hasGameId' ],
-  'game/ready': [ 'isLoggedIn', 'isInGame' ],
+  'game/spectate/join': [ 'isLoggedIn', 'hasGameId' ],
+  'game/spectate/leave': [ 'isLoggedIn', 'hasGameId' ],
+  'game/spectate': [ 'isLoggedIn' ],
+  'game/ready': [ 'isLoggedIn', 'hasGameId', 'isInGame' ],
   'game/set-is-ranked': 'isInGame',
-  'game/leave-lobby': [ 'isSocket', 'isLoggedIn', 'isInGame' ],
+  'game/leave-lobby': [ 'isSocket', 'isLoggedIn', 'hasGameId' ],
```

Comment:
TODO: need to review the endpoints that used to require `isInGame` or that logically should require it and ensure they each verify that the user is in fact in the game.